www/placeholders/new.php

```diff
@@ -0,0 +1,67 @@
+<?
+use function Safe\session_unset;
```

Comment:
Let's move this file to `/ebook-placeholders/new` for consistency

Reply:
Done.